src/md.cc

```diff
@@ -400,7 +405,47 @@ void MGmol<OrbitalsType>::md(OrbitalsType** orbitals, Ions& ions)
     if (ct.restart_info < 3)
     {
         double eks = 0.;
-        quench(**orbitals, ions, ct.max_electronic_steps, 20, eks);
+        bool do_quench = true;
+#ifdef MGMOL_HAS_LIBROM
+        if (ct.getROMOptions().rom_stage == ROMStage::ONLINE_MVP)
+        {
+            ExtendedGridOrbitals** extended_orbitals = reinterpret_cast<ExtendedGridOrbitals**>(orbitals);
+            (*extended_orbitals)->set(ct.getROMOptions().basis_file, ct.numst); 
+            (*extended_orbitals)->orthonormalizeLoewdin();
+            (*extended_orbitals)->setDataWithGhosts(true);
+            (*extended_orbitals)->setIterativeIndex(10);
+
+            std::shared_ptr<ProjectedMatricesInterface> projmatrices
+                = getProjectedMatrices();
+            projmatrices->setDMuniform(ct.getNelSpin(), 0);
+            projmatrices->printDM(os_);
+
+            std::vector<double> positions;
+            getAtomicPositions(positions);
+            std::vector<short> anumbers;
+            getAtomicNumbers(anumbers);
+            PinnedH2O H2O_molecule;
+            H2O_molecule.rotate(positions, anumbers);
+
+            ions.setPositions(positions, anumbers);
+            moveVnuc(ions);
```

Comment:
call to moveVnuc() should be replaced by something like:
`Potentials& pot = hamiltonian_->potential();
pot.initialize(ions);`